src/components/layout/Header.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.header`
+  color: #FFFFFF;
+  background-color: var(--color-header-bg);
+  min-height: var(--header-height);  
+  font-size: 1.5am;
+  //position: relative;
+
+  nav {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav-links {
+    flex: 1;
+    display: flex;
+    justify-content: flex-end;
+
+    list-style-type: none;
+    margin-right: 3rem;
+    white-space: nowrap;
+
+    a {
+      color: white;
+      padding: 0.8em;
+    }
+  }
+
+  .logo,.connect-wallet {
+    padding: 1rem;
+    margin: 1rem;
+    border: 2px solid var(--color-btn-border);
+    color: var(--color-btn-border);
+    vertical-align: middle;
+
+    &:hover {
+      color: white;
+      border-color: white;
+      cursor: pointer;
+    }
+  }
+
+  .logo {
+    font-size: 1.2em;
```

Comment:
Why are we sometimes using rem and other times em?

Reply:
The logo is a placeholder. So it will replace by an image as soon as we have a logo.

rem and em are using depending on "in relation to what" you want to define the size. A global reference, or a relative (current paragraph)

They both have it's use